browser/brave_browser_main_parts.cc

```diff
@@ -76,6 +76,17 @@ void BraveBrowserMainParts::PostBrowserStart() {
         tor_legacy_path, base::DoNothing(),
         ProfileMetrics::DELETE_PROFILE_SETTINGS);
   }
+  for (Profile* profile : profile_manager->GetLoadedProfiles()) {
+    const base::FilePath tor_legacy_session_path =
+        profile->GetPath()
+            .AppendASCII("session_profiles")
```

Comment:
How about making `session_profiles` as a constant instead of hard-cording?
It looks like some places used hard coded string.

Reply:
This is only used for migration

Reply:
It also hard coded below.
```
bool IsSessionProfilePath(const base::FilePath& path) {
  return path.DirName().BaseName() ==
         base::FilePath(FILE_PATH_LITERAL("session_profiles"));
}
```

Reply:
fixed in c310e8d4f0221c3111f23972c48b4baecced8d13